picasso/src/main/java/com/squareup/picasso3/Utils.java

```diff
@@ -336,8 +337,7 @@ static void flushStackLocalLeaks(Looper looper) {
   }
 
   static class PicassoThreadFactory implements ThreadFactory {
-    @SuppressWarnings("NullableProblems")
-    public Thread newThread(Runnable r) {
+    @Override public Thread newThread(@NonNull Runnable r) {
```

Comment:
we've been removing NonNulls in favor of defaults, and only marking Nullable parameters.

Reply:
IntelliJ gives a warning here cuz the supertype parameter is marked/known non-null.